client/src/components/lobby/GameCreator.tsx

```diff
@@ -71,10 +71,10 @@ export function GameCreator(props: GameCreatorProps) {
             onClick={async () => {
               if (!isLoading) {
                 setError("");
-                if (!Token.isGoogleToken(playerToken) && visibility !== "local") {
-                  setError("Google sign-in is required to create a match");
-                  return;
-                }
+                // if (!Token.isGoogleToken(playerToken) && visibility !== "local") {
```

Comment:
did you mean to leave these commented out?

Reply:
oops nice catch - thanks 🙏